packages/contracts-bedrock/justfile

```diff
@@ -27,6 +27,10 @@ forge-build-dev *ARGS:
 build-source:
   forge build --skip "/**/test/**" --skip "/**/scripts/**"
 
+# Builds source contracts and scripts, skipping tests.
+build-no-tests:
+  forge build --skip "/**/test/**"
```

Comment:
What is the use case of this? Would be nice to add this as a comment

Reply:
Useful for OPCM where you need to build everything except the tests (incl. scripts)